test/unit/dom-repeat-elements.html

```diff
@@ -367,6 +367,30 @@
   });
 </script>
 
+<dom-module id="x-repeat-filter-and-sort-by-nested-property">
+    <template>
+      <template id="repeater" is="dom-repeat" items="{{items}}">
+        <x-foo itema-prop="{{item.prop.nestedProp}}"></x-foo>
+      </template>
+    </template>
+    <script>
+    Polymer({
+      is: 'x-repeat-filter-and-sort-by-nested-property',
+      properties: {
+        items: {
+          value: [
```

Comment:
This should be `function() { return [...]; }`

Reply:
May I use ES2015 style `value () { return [] }`?